docs/status-bar.md

```diff
@@ -2,4 +2,153 @@
 id: status-bar
 title: Different status bar configuration based on route
 sidebar_label: Different status bar configuration based on route
----
\ No newline at end of file
+---
+
+If you don't have a navigation bar, or your navigation bar changes color based on the route, you'll want to ensure that the correct color is used for the content.
+
+## StackNavigator and DrawerNavigator
+
+This is a simple task when using the StackNavigator or DrawerNavigator. You can simply render the `StatusBar` component, which is exposed by React Native, and set your config.
+
+```javascript
+class Screen1 extends React.Component {
+  render() {
+    return (
+      <SafeAreaView style={[styles.container, { backgroundColor: '#6a51ae' }]}>
+        <StatusBar
+          barStyle="light-content"
+          backgroundColor="#6a51ae"
+        />
+        <Text style={[styles.paragraph, { color: '#fff' }]}>
+          Light Screen
+        </Text>
+        <Button
+          title="Next screen"
+          onPress={() => this.props.navigation.navigate('Screen2')}
+          color={isAndroid ? "blue" : "#fff"}
+        />
+      </SafeAreaView>
+    );
+  }
+}
+
+class Screen2 extends React.Component {
+  render() {
+    return (
+      <SafeAreaView style={[styles.container, { backgroundColor: '#ecf0f1' }]}>
+        <StatusBar
+          barStyle="dark-content"
+          backgroundColor="#ecf0f1"
+        />
+        <Text style={styles.paragraph}>
+          Dark Screen
+        </Text>
+        <Button
+          title="Next screen"
+          onPress={() => this.props.navigation.navigate('Screen1')}
+        />
+      </SafeAreaView>
+    );
+  }
+}
+```
+
+```javascript
+export default StackNavigator({
+  Screen1: {
+    screen: Screen1,
+  },
+  Screen2: {
+    screen: Screen2,
+  },
+}, {
+  headerMode: 'none',
+});
+```
+
+![StackNavigator with different StatusBar configs](./assets/statusbar/statusbar-stack-demo.gif)
+
+```javascript
+export default DrawerNavigator({
+  Screen1: {
+    screen: Screen1,
+  },
+  Screen2: {
+    screen: Screen2,
+  },
+});
+```
+
+![DrawerNavigator with different StatusBar configs](./assets/statusbar/statusbar-drawer-demo.gif)
+
+## TabNavigator
+
+If you're using a TabNavigator it's a bit more complex because the screens on all of your tabs are rendered at once - that means that the last `StatusBar` config you set will be used (likely on the final tab of your tab navigator, not what the user is seeing).
```

Comment:
yeah this is very unfortunate. looking forward to integrating status bar style into react-nav itself